apps/prairielearn/src/sprocs/sync_assessments.sql

```diff
@@ -566,11 +566,13 @@ BEGIN
         AND a.course_instance_id = syncing_course_instance_id
         AND (da.errors IS NOT NULL AND da.errors != '');
 
-    -- Ensure all assessments have an assessment module, default number=0.
+    -- Ensure all assessments have an assessment module. We'll use the "Default"
+    -- module if one is not specified. The assessment module syncing code will
+    -- ensure that such a module exists.
     UPDATE assessments AS a
     SET
         assessment_module_id = COALESCE(a.assessment_module_id,
-            (SELECT id FROM assessment_modules WHERE number = 0 AND course_id = syncing_course_id))
+            (SELECT id FROM assessment_modules WHERE name = 'Default' AND course_id = syncing_course_id))
```

Comment:
Curiously, this more closely matches what the docs list...

https://github.com/PrairieLearn/PrairieLearn/blob/a7bed786b11dfd79ea92964e2e73207b8c41fbb9/docs/assessment.md?plain=1#L39

Reply:
This was also the intention of the existing code, which would correctly do that if `"Default"` was created automatically (we explicitly set `number = 0` in that case). But yes, the new code is definitely more correct!